setup.py

```diff
@@ -2,18 +2,15 @@
 from os import path
 from io import open
 
-import versioneer
-
 here = path.abspath(path.dirname(__file__))
 
 # Get the long description from the README file
 with open(path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
 setup(name='pynvml',
-      version=versioneer.get_version(),
-      cmdclass=versioneer.get_cmdclass(),
-      python_requires='>=3.6',
```

Comment:
Think it is unwise to drop this minimum. It is unlikely their code works on anything older than Python 3.6 (possibly it even needs a newer version). The code we still ship at least needs Python 3.6 (possibly newer depending on what we test with)